src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAroundCheck.java

```diff
@@ -291,6 +410,19 @@
  *   &lt;property name=&quot;ignoreEnhancedForColon&quot; value=&quot;true&quot; /&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>Example:
+ * </p>
+ * <pre>
+ * class Test {
+ *     public static void main(String[] args) {
+ *         int a=4; // 2 violations , '=' is not followed and preceded by whitespace.
+ *         char[] vowels = {'a', 'e', 'i', 'o', 'u'};
+ *         for (char item: vowels) { // ok
```

Comment:
please add this FOR `for (char item: vowels) {` in some other example  to show that we violate it, but this option allow void violation.

Please do the same for all other `allowXxxxxx` cases. We need to show at least in default config that we violate such code.

Reply:
Hello @romani i cannot add an example of foreach colon violation as the default config of ignoreEnhancedForColon is true. 
Can u please elaborate on this a more 
Sorry for the trouble

Reply:
ok, in this case we need example of `ignoreEnhancedForColon=false` please do this all others that are ignored by default.
We need to show each property in action for activated and disabled mode, to let user see how behavior is changing then value of it is changing.

Reply:
Hey @romani Can you just confirm that this is what you want?
This is the default config with set of examples I will add more to it and do the changes accordingly 
```
class Test {
    public Test(){} // 2 violations , '{' is not followed and preceded by whitespace.
    public static void main(String[] args) {
        if (foo) { // ok
            // body
        }
        else{ // violation
            // body
        }
        for (int i = 1; i &gt; 1; i++) {} // 2 violations ,
                                            // '{' is not followed and preceded by whitespace.
        Runnable noop = () -&gt; {}; // 2 violations ,
                                        // '{' is not followed and preceded by whitespace.
    }
}
```

Now if i set a config with `allowEmptyConstructors = true` my example should look like this 
```
class Test {
    public Test(){} // ok
    public static void main(String[] args) {
        if (foo) { // ok
            // body
        }
        else{ // violation
            // body
        }
        for (int i = 1; i &gt; 1; i++) {} // 2 violations ,
                                            // '{' is not followed and preceded by whitespace.
        Runnable noop = () -&gt; {}; // 2 violations ,
                                        // '{' is not followed and preceded by whitespace.
    }
}
```
So, you basically, want me to give examples for the default config, and the complement of it as well (i.e. for default: I give an example, and provide another similar example for something that's not a default ), is that right? 
Is that all that you want me to do?

Reply:
@romani please check this one i added the for loop in the default config too where ignoreEnhancedForColon is true and made a config where ignoreEnhancedForColon is false with an example
And i think all the examples are covered, please tell me if I miss something else 
Sorry for the trouble

Reply:
Ok I see it now in code for default.